lib/rock_paper_scissors/MovesLogic.rb

```diff
@@ -0,0 +1,10 @@
+module MovesLogic
+
+  MOVES = ["rock", "paper", "scissors"]
+
+  WIN_MATRIX = [["It's a draw", "You won", "You lose"], ["You lose", "It's a draw", "You won"], ["You lose", "You won", "It's a draw"]]
```

Comment:
This is not a pretty solution like we discussed. How would it change if you add the 4th element, "a well"? Which has the same meaning as the rock.

How about creating a separate class for each element, and specify the elements it beates/beaten? For example:
```
class Rock
  @beats = [Scissors]

  def beats?(el)
    @beats.include?(el)
  end
end
```
It's a pseudo-code but you get the idea